src/parcel.js

```diff
@@ -26,7 +26,8 @@ export default class Parcel extends React.Component {
     this.addThingToDo('mount', () => {
       const mountParcel = this.props.mountParcel || this.mountParcel
       if (!mountParcel) {
-        throw new Error(`<Parcel /> was not passed a mountParcel prop, nor is it rendered where mountParcel is within the React context`)
+        throw new Error(`<Parcel /> was not passed a mountParcel prop, nor is it rendered where mountParcel is within the React context.
+					If you are using <Parcel /> within a module that is not using single-spa, you may need to import mountRootParcel from single-spa and pass it into the mountParcel prop`)
```

Comment:
I'd suggest changing this to something more like the following:
`If you are using <Parcel /> within a module that is not a single-spa application, you will need to import mountRootParcel from single-spa and pass it into <Parcel /> as a mountParcel prop`

I'm confident this same issue would occur if you we're trying to use <Parcel /> inside the single-spa-config/root application